content/docs/composition-vs-inheritance.md

```diff
@@ -45,11 +45,11 @@ function WelcomeDialog() {
 }
 ```
 
-**[Try it on CodePen](https://codepen.io/gaearon/pen/ozqNOV?editors=0010)**
+**[Probeer het op CodePen](https://codepen.io/gaearon/pen/ozqNOV?editors=0010)**
 
-Anything inside the `<FancyBorder>` JSX tag gets passed into the `FancyBorder` component as a `children` prop. Since `FancyBorder` renders `{props.children}` inside a `<div>`, the passed elements appear in the final output.
+Eender wat binnen de `<FancyBorder>` JSX tag staat zal doorgegevn worden in de `FancyBorder` component als een `children` prop. Aangezien `FancyBorder` `{props.children}` rendert binnen een `<div>` zullen de doorgegevn elementen verschijnen in de finale uitvoer.
```

Comment:
Eender wat binnen => Iets wat binnen (eender is beetje oubollig)
zal doorgegevn worden in de => wordt doorgegeven aan het
zullen => , zullen (komma voor zullen)
finale => uiteindelijke
doorgegevn => doorgegegeven